climada/engine/impact.py

```diff
@@ -1372,7 +1372,147 @@ def _selected_exposures_idx(self, coord_exp):
             LOGGER.warning("No exposure coordinates match the selection.")
         return sel_exp
 
+    @classmethod
+    def concat(cls, imp_list,concat_type='time_window'):
+        """Concatenate impact objects of the same hazard type, frequency, 
+        exposure, and coordinate system.
 
+        Parameters
+        ----------
+        imp_list : list of climada.engine.Impact objects
+                list of Impact object to concatenate
+        concat_type: string
+                string specifying the type of concatenation to be done.
+                    'time_window': concatenate impacts for different time windows / time slices within a year (e.g. May to June, July to August,...)
+                    'exposure': concatenate impacts with separate exposures
+        Returns
+        --------
+        impact: climada.engine.Impact
+               New impact object which is a concatenation of all impacts
+        """
+       
+        if len(imp_list) == 0:
+           return cls()
+       
+        if concat_type not in ['time_window','exposure']:
+           raise ValueError(f"Concatenation option {concat_type} not available.")
```

Comment:
I suppose @emanuel-schmid would like you to use an [Enum](https://docs.python.org/3/library/enum.html) class here. C.f. for instance https://github.com/CLIMADA-project/climada_python/blob/main/climada/util/lines_polys_handler.py  .

Reply:
Hm...not sure if I understand what you mean :-)

Reply:
When you have a function that uses different choices, it is best to use an `Enum` class object to define these choices. Thus, instead of using strings to label your choices, you use objects of an `Enum` class.

Reply:
Unfortunately, I'm still not sure if it makes sense in this case an how to use it...

Reply:
resolved because I removed the concatenation option "exposure" as it is not implemented anyways.